Jenkinsfile

```diff
@@ -3,14 +3,16 @@ node('docker && imgtec') {
     docker_image = docker.image "imgtec/creator-builder:latest" // TODO for now using the creator one
     docker_image.inside {
         stage('Prepare') {
+            deleteDir() // clean up the workspace to save space
```

Comment:
This seems to be causing build issues. Perhaps just better to go with the clean checkout, instead of 'checkout scm' you can use a full custom one:
```
            checkout([$class: 'GitSCM',
                userRemoteConfigs: scm.userRemoteConfigs,
                branches: scm.branches,
                doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
                submoduleCfg: scm.submoduleCfg,
                browser: scm.browser,
                gitTool: scm.gitTool,
                extensions: scm.extensions + [
                    [$class: 'CleanCheckout'],
                    [$class: 'PruneStaleBranch'],
                ],
            ])
```